content/en/catalyst/fund9/challenges/developer-ecosystem/campagin-brief.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Develper Ecosystem"
+linkTitle: "Develper Ecosystem"
+type: docs
+hide_feedback: true
+description: "How can we create a positive developer experience that helps the developer focus on building successful apps?"
+---
+<img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/01-Developer-Ecosystem-4d9cc4.png" style="width:500px;height500px">
+
+Fund Size: **1,000,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/398985) and [browse ideas]().
```

Comment:
Do we need to change the campaign link to this link, 'https://cardano.ideascale.com/c/campaigns/26597/about'?